unlock-app/src/components/content/event/LandingPage.tsx

```diff
@@ -0,0 +1,226 @@
+import { Disclosure } from '@headlessui/react'
+import { Button } from '@unlock-protocol/ui'
+import { BsPlusLg as PlusIcon } from 'react-icons/bs'
+import { Icon } from '@unlock-protocol/ui'
+import Image from 'next/image'
+import Link from 'next/link'
+
+interface AccordionProps {
+  title: string
+  children: React.ReactNode
+}
+
+const Accordion = ({ title, children }: AccordionProps) => {
+  return (
+    <Disclosure>
+      {() => (
+        <>
+          <Disclosure.Button className="border-t flex w-full py-2 text-left border-gray-600">
```

Comment:
```suggestion
          <Disclosure.Button className="border-t flex w-full py-2 text-left border-gray-600 justify-between">
```
`+` icons are not aligned when the title length is different, lets keep the plus icon at the end?